_dev/src/assets/scss/base/_typography.scss

```diff
@@ -3,6 +3,10 @@
   line-height: 1.4em;
 }
 
+#configuration {
+  max-width: none!important;
```

Comment:
If I remember well, @amaury-hanser had set a max width that you could disable instead of override